### PR TITLE
Don't use --force-install unless it's available

### LINKED ISSUE #86: --force-install addition to compile.sh isn't supported by vmware-install.pl
Commit 3c8946b95cec12c0748722bc18796af9618030b4 introduced a `--force-install` parameter to the compile.sh which vmware-install.pl does not appear to support, causing it to emit a `Usage:` block indicating an error. The vmware-install.pl run fails.

```
...
 60400K .......... .......... .......... .......... .......... 99% 1.64M 0s
 60450K .......... .......... .......... .......... .......... 99% 1.64M 0s
 60500K .......... .......... .......... .......... .......... 99% 2.49M 0s
 60550K .......... .......... .......... ..........           100% 1.74M=35s

2015-12-22 16:43:17 (1.67 MB/s) - ‘com.vmware.fusion.tools.linux.zip-7.1.1.tar’ saved [62044160/62044160]

mount: block device /tmp/vmware-tools-patches/downloads/payload/linux.iso is write-protected, mounting read-only
initctl: Job failed while stopping
initctl: Unknown instance:
VMware Tools 9.9.2 build-2496486 for Linux installer
Usage: ./vmware-install.pl [[-][-]d[efault]]
default: Automatically answer questions with the proposed answer.
[[-][-]prefix=<path to install product: bin, lib, doc>]    Put the installation
at <path> instead of the default location.  This implies '--default'.
--clobber-kernel-modules=<module1,module2,...>    Forcefully removes any VMware
related modules installed by any other installer and installs the modules
provided by this installer.  This is a comma seperated list of modules.
...
```

### PR DESCRIPTION
**Pull request by sykesm**
Fixes #86 and #87.

### CODE REVIEW

File: compile.sh [hunk 1]
```diff
@@ -33,6 +33,10 @@ fi
 
 pushd vmware-tools-distrib >/dev/null
 
-sudo ./vmware-install.pl --default --force-install ${VMWARE_INSTALL_OPTIONS}
+if sudo ./vmware-install.pl --help 2>&1 | grep -q 'force-install'; then
```
Comment (sykesm) [inline compile.sh]: Without `sudo`, the `vmware-install.pl` script would simply abort with a message about running the program as a super user.

**Comment by mikeputnam** [comment]
I tested this with our configuration and this worked.  :+1:

**Comment by mafrosis** [comment]
+1 @sykesm 

@rasa please merge this!

**Comment by rasa** [comment]
Sorry for the delay. Thanks for the patch!

**Pull request merged**